### ISSUE TITLE
我在使用ernie-3.0-medium-zh模型进行句子相似度计算，发现中国和美国的余弦距离是0.98。我是做错了什么了吗？

### ISSUE THREAD

**Issue by glacierck** [PaddlePaddle/PaddleNLP#6372]
`class PaddleNLP():
    def __init__(self, model: str = "ernie-3.0-medium-zh"):
        self.model = AutoModel.from_pretrained(model)
        self.model.eval()

        self.tokenizer = AutoTokenizer.from_pretrained(model, use_fast=True)
        if not self.tokenizer.pad_token:
            self.tokenizer.pad_token = "<pad>"
        self.__dimension = None

    def to_embeddings(self, data, **_):
        if not isinstance(data, list):
            data = [data]
        inputs = self.tokenizer(
            data
            , padding=True
            , truncation=True
            , return_tensors="pd"
        )

        outs = self.model(**inputs)[0]
        emb = self.post_proc(outs, inputs).squeeze(0).detach().numpy()

        return np.array(emb).astype("float32")

    def post_proc(self, token_embeddings, inputs):
        attention_mask = paddle.ones(inputs["token_type_ids"].shape)
        input_mask_expanded = (
            attention_mask.unsqueeze(-1).expand(token_embeddings.shape).astype("float32")
        )
        sentence_embs = paddle.sum(
            token_embeddings * input_mask_expanded, 1
        ) / paddle.clip(input_mask_expanded.sum(1), min=1e-9)
        return sentence_embs

    @property
    def dimension(self):
        if not self.__dimension:
            self.__dimension = len(self.to_embeddings("foo"))
        return self.__dimension

    def calculate_similarity(self, document1, document2):
        vec1 = self.to_embeddings(document1).reshape(1, -1)  # 调整维度以适应cosine_similarity函数输入要求, 一行一列
        vec2 = self.to_embeddings(document2).reshape(1, -1)

        sim_score = cosine_similarity(vec1, vec2)[0][0]

        return sim_score

if __name__ == "__main__":
    encoder = PaddleNLP(model='ernie-3.0-medium-zh')
    sim_score = encoder.calculate_similarity("中国",  "美国")
    print(sim_score)   # 0.9841709136962891
`

**Comment by tianchiguaixia** [comment]
很正常，使用bert都是0.9以上。你得使用sentence-bert

**Comment by w5688414** [comment]
或者自己标注数据训练一下
